src/iop/ashift_lsd.c

```diff
@@ -1914,7 +1914,7 @@ static int reduce_region_radius( struct point * reg, int * reg_size,
                                  image_char used, image_double angles,
                                  double density_th )
 {
-  double density,rad1,rad2,rad,xc,yc;
+  double density,radius1,radius2,rad,xc,yc;
```

Comment:
rad1 and rad2 are unfortunately already defined numeric constants in [dlgs.h](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646951(v=vs.85).aspx) - this change is basically a naming collision fix